scripts/migration/database-down

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
These database scripts are used to orchestrate liquibase migrations locally.  No need to change them and can just be copied between repos.